testhelpers/chain.go

```diff
@@ -171,9 +171,13 @@ func MustNewTipSet(blks ...*types.Block) types.TipSet {
 	return ts
 }
 
+type requirePutTsasChainStore interface {
+	PutTipSetAndState(context.Context, *chain.TipSetAndState) error
+}
+
 // RequirePutTsas ensures that the provided tipset and state is placed in the
 // input store.
-func RequirePutTsas(ctx context.Context, t *testing.T, chn chain.Store, tsas *chain.TipSetAndState) {
+func RequirePutTsas(ctx context.Context, t *testing.T, chn requirePutTsasChainStore, tsas *chain.TipSetAndState) {
```

Comment:
Is this really necessary? It can be inlined as a one-liner, which would eliminate the need for the interface.

Reply:
I wasn't aware interface definitions could be inlined. How do you do that?

Reply:
I mean inline `RequirePutTsas`. The name isn't great in the first place and it doesn't save us much.